<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="Add">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="SetAlarm">
      <property name="text">
       <string>设置闹钟</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Minus">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Tranform">
      <property name="text">
       <string>切换</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="TimeCal">
      <property name="text">
       <string>计时器</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Flag">
      <property name="text">
       <string>标记</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Stop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Reset">
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Start">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
